### PR TITLE
Add descriptive status info text

### PR DESCRIPTION
**Pull request by mareklibra**
descriptiveText

### CODE REVIEW

File: src/config/constants.ts [hunk 1]
```diff
@@ -39,3 +39,16 @@ export const HOST_STATUS_LABELS: { [key in Host['status']]: string } = {
   installed: 'Installed',
   error: 'Error',
 };
+
+export const HOST_STATUS_DETAILS: { [key in Host['status']]: string } = {
+  discovering: 'Capabilities of the host are being collected.',
```
Comment (mareklibra) [inline src/config/constants.ts:44]: @andybraren, can you please have a look at these texts? Will be rendered on click on host-status cell, supplemented by message from backend.

Comment (andybraren) [reply src/config/constants.ts:44]: I suspect we'll need to tweak this some more, but here's a first pass based on the states documented [here](https://docs.google.com/document/d/1jxNMTlotmJ0GFZ1GUEQ3hfOOVLzMT6mvY8Ufo5RdErY/edit#heading=h.gptlbp92g8lj). FYI @avishayt

**Discovering**
This host is transmitting its hardware and networking information to the installer. Please wait while this information is received.

**Known** ([I wonder](https://docs.google.com/document/d/1jxNMTlotmJ0GFZ1GUEQ3hfOOVLzMT6mvY8Ufo5RdErY/edit?disco=AAAAGlR2YNc) if this should become `Ready`)
This host meets the minimum hardware and networking requirements and will be included in the cluster.

**Disconnected**
This host has lost its connection to the installer and will not be included in the cluster unless connectivity is restored.

**Insufficient**
This host does not meet the minimum hardware or networking requirements and will not be included in the cluster.

**Disabled**
This host was manually disabled and will not be included in the cluster. Enable this host to include it again.

**Installed** ([I wonder](https://docs.google.com/document/d/1jxNMTlotmJ0GFZ1GUEQ3hfOOVLzMT6mvY8Ufo5RdErY/edit?disco=AAAAJn0uG3k) if this should become `Provisioned`)
This host completed its installation successfully.

Comment (avishayt) [reply src/config/constants.ts:44]: Text looks great, you are missing "Error" state

Comment (mareklibra) [reply src/config/constants.ts:44]: Error: 
This host failed its installation.

And additional info will follow and/or will be included within events.

**Pull request merged**